### DISCUSSION TITLE
Add highlighting to lines in markdown editor in block code

### DISCUSSION THREAD

**Discussion by oim37** [wintercms/wn-blog-plugin#55] [Q&A]
Hello,

I read manual about markdown code block and not find option where i can highlight lines in code block.
1. To add content in code block i use

![image](https://github.com/wintercms/wn-blog-plugin/assets/6842168/d3abf6d0-d578-4a13-81b9-752cc4222ec4)

In second example lines marked, after save and close it's ok, but when i open post to edit some text \&lt; automaticly changed to left quote and not showing part of content.
And when there are a lot of left quotes, they are all converted from \&lt; to < and i need to change everything again manually. 

In the first example, it considers them as code and displays them as needed, but I cannot select lines there. Asterisks and other attempts to highlight a line did not work for me.

I want something like this:
![image](https://github.com/wintercms/wn-blog-plugin/assets/6842168/3c332b61-a3e0-4812-831a-e394aa3a44ce)
![image](https://github.com/wintercms/wn-blog-plugin/assets/6842168/939a91aa-92f6-4c74-b59d-c89f88dd50f0)

I searching and find this, as i understand this 3rd party uses by Winter CMS: 
i think it what i need https://github.com/thephpleague/commonmark/discussions/631 but i don't understand how make it and it some years ago is it actual or no..

Maybe someone has already encountered this and there is a convenient solution, since I don’t want to change markdown editor, overall I really liked it and found it comfortable except for this moment.

**Comment by LukeTowers** [comment]
@bennothommo would you be willing to take a look at this?
